tests/distributed/test_workspaces/test_remote_workspaces.py

```diff
@@ -146,6 +146,19 @@ def test_workspace_delete():
         assert client.workspaces.delete(workspace_id)
 
 
+def test_workspace_clear():
+    client = JinaDClient(host=__default_host__, port=8000)
+    workspace_id = client.workspaces.create(
+        paths=[os.path.join(cur_dir, 'empty_flow.yml')]
+    )
+    assert DaemonID(workspace_id).type == 'workspace'
+    assert(
+        WorkspaceItem(**client.workspaces.get(id=workspace_id)).state
+        == RemoteWorkspaceState.ACTIVE
+    )
+    assert client.workspaces.clear()
```

Comment:
Here's what the test can cover - 
1. create >1 workspaces
2. `assert client.workspaces.clear()`
3. asset no workspaces exist using `list()`

Reply:
Hi deepankarm, `assert workspace_id in client.workspaces.list()` this line is to be 
added ?

Reply:
We are working on re-enabling CI for external contributors. I'll update here, once that is ready.

Reply:
Ok @deepankarm , kindly let me know after re-enabling it.